subworkflows/local/transdecoder.nf

```diff
@@ -0,0 +1,27 @@
+//
+// TRANSDECODER SUBWORKFLOW
+//
+
+include { TRANSDECODER_LONGORF as LONGORF } from '../../modules/nf-core/transdecoder/longorf/main'
+include { TRANSDECODER_PREDICT as PREDICT } from '../../modules/nf-core/transdecoder/predict/main'
+
+workflow TRANSDECODER {
+    take:
+    contigs       // channel: [ val(meta), [ contigs ] ]
+
+    main:
+    ch_versions  = Channel.empty()
+
+    LONGORF (contigs)
+    PREDICT (contigs, LONGORF.out.folder)
+    ch_versions = ch_versions.mix(LONGORF.out.versions)
```

Comment:
While I see they're from the same tool, technically this is missing the version from `PREDICT`.